### ISSUE TITLE
Usersnap Feedback - Edit to title does not show up on card view. I changed the title of the fi[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#694]
**Sender**: scottowenfletcher@gmail.com
**Comment**: Edit to title does not show up on card view. I changed the title of the first entry (will upload screenshot of reader view below)
[Open #393 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-201-a37e4cf6/393)

<a href='https://usersnappublic.s3.amazonaws.com/2019-05-27/17-02/f440a46f-e33e-402b-b906-364bf59ef384.png'>![Usersnap Feedback - Edit to title does not show up on card view. I changed the title of the fi(...)](https://usersnappublic.s3.amazonaws.com/2019-05-27/17-02/6e26a2ad-89ee-43e5-b3b6-b5423de713a3.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-05-27/17-02/f440a46f-e33e-402b-b906-364bf59ef384.png'>Download original image</a>

**Browser**: Chrome 74 (Windows 10)
**Referer**: [https://participedia.xyz/?query=milan](https://participedia.xyz/?query=milan)
**Screen size**: 1280 x 720 **Browser size**: 1920 x 867

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by scottofletcher** [comment]
![image](https://user-images.githubusercontent.com/29512491/58427930-e1c6fa00-8055-11e9-9985-5a5708d0caa1.png)

**Comment by ascott** [comment]
i believe this is because of the 5 min lag on search indexing. without the search query, the title does show up with the edited value. @dethe is that correct about the search lag time?

**Comment by dethe** [comment]
Yes, we store title, description, etc with search results, so we don't need additional queries per search result.

I could fix this at the cost of making searches a little more complex and slightly slower (though it would speed up search indexing and make search table smaller, so maybe a net win?

**Comment by jesicarson** [comment]
oh i see. 
hmm. well it seems that it might need to be addressed, but lets wait until more people care about it maybe? if it ends up feeling like a bug to a lot of people, we might want to deal. but i'm not sure if everyone will immediately go and search for a case they just entered.
removing p1 we can come back to this later thanks!

**Comment by scottofletcher** [comment]
yeah doesn't sound like it's a pressing issue. it's not so much that they will be searching for the entry they just edited, but that they (like me) will see it on the homepage (since recently updated content goes to the top) and assume their edits weren't saved
